components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -353,6 +354,25 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
       newMoreInfoRow.insert(varMoreInfoInner);
       variantFooter.insert({before : newMoreInfoRow});
 
+      // restrict only numerical input for start_position and end_position variant fiellds
+      newMoreInfoRow.select('input:[id$=_start_position]', 'input:[id$=_end_position]').each( function(input) {
+        input.observe('keydown', function(evt){
+          var theEvent = evt || window.event;
+          var key = theEvent.keyCode || theEvent.which;
+          key = String.fromCharCode( key );
+          var regex = /[0-9]|\./;
+          if( !regex.test(key) ) {
+            theEvent.returnValue = false;
+            if(theEvent.preventDefault) theEvent.preventDefault();
+          }
+        });
+      });
```

Comment:
This disallows ALL keys that are not numbers. It's impossible to use BACKSPACE for corrections, CTRL+V for pasting a value from a lab report, TAB for navigating to the next control etc, which makes the input quite unusable.